package.json

```diff
@@ -78,7 +78,7 @@
   "author": "Unbxd",
   "license": "ISC",
   "dependencies": {
-    "@unbxd-ui/unbxd-search-core": "0.2.7",
+    "@unbxd-ui/unbxd-search-core": "^0.2.8",
```

Comment:
we can use the exact version here right so that we can be sure of the changes we are using from search js-core ?

Reply:
yea